### PR TITLE
Update EnoughTokenValidator

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

**Comment by Marik-D** [comment]
Requested changes so the bot doesn't merge PR right away. We should probably make the automatic merging opt-in only. For example with a special label: `auto-merge`

File: universal-login-relayer/lib/integration/ethereum/validators/EnoughTokenValidator.ts [hunk 1]
```diff
@@ -10,24 +10,24 @@ const isContract = async (provider: providers.Provider, contractAddress: string)
   return isContractExist(bytecode);
 };
 
-export const hasEnoughToken = async (gasToken: string, walletContractAddress: string, gasLimit: utils.BigNumberish, provider: providers.Provider) => {
+export const hasEnoughToken = async (gasToken: string, walletContractAddress: string, gasLimit: utils.BigNumberish, gasPrice: utils.BigNumberish, provider: providers.Provider) => {
```
Comment (marekkirejczyk) [inline universal-login-relayer/lib/integration/ethereum/validators/EnoughTokenValidator.ts]: gasToken, gasLimit, gasPrice -> PaymentOptions

Comment (marekkirejczyk) [inline universal-login-relayer/lib/integration/ethereum/validators/EnoughTokenValidator.ts]: Consider two additional refactorings:
- extract isContract to separate files
- move EnoughTokenValidator to commons

File: universal-login-commons/lib/core/utils/contracts/contractHelpers.ts [hunk 2]
```diff
@@ -11,4 +11,10 @@ export const isContractExist = (bytecode: string) => {
   return bytecode !== '0x';
 };
 
+export const isContract = async (provider: providers.Provider, contractAddress: string) => {
+  // TODO: Only whitelisted contracts
```
Comment (marekkirejczyk) [inline universal-login-commons/lib/core/utils/contracts/contractHelpers.ts]: Misleading comment

**Pull request merged**